Cargo.lock

```diff
@@ -5097,12 +5097,12 @@ dependencies = [
  "anyhow",
  "apache-avro 0.17.0",
  "arrow-arith 50.0.0",
- "arrow-array 50.0.0",
- "arrow-buffer 50.0.0",
+ "arrow-array 48.0.1",
+ "arrow-buffer 48.0.1",
```

Comment:
Why icelake is affected? I only ran `cargo update -p arrow-udf-js`

Reply:
`icelake` specifies `arrow >=48`. Rare case. 😄

https://github.com/icelake-io/icelake/blob/54fd72fbd1dd8c592f05eeeb79223c8a6a33c297/Cargo.toml#L15-L23

Reply:
Because icelake depends on `>=48`, and we depends only multiple versions of arrow. So this is indeed valid...

Reply:
cc @liurenjie1024 `>=48` seems not to always work. Can we specify a fixed major version to conform the semantics versioning?

Reply:
Manually reverted this change.